### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by W00t3k**
Add largest Cellular Hacking resource on the Internet

### CODE REVIEW

**Comment by cat998** [comment]
已收到！

**Review by W00t3k — COMMENTED** [review]
Largest Cellular Hacking resource on the Internet

**Review by its0x08 — APPROVED** [review]

**Review by W00t3k — COMMENTED** [review]
Thank you!

**Review by W00t3k — COMMENTED** [review]
Thank you

**Review by pimpthehoez — APPROVED** [review]

**Review by Perla2411 — APPROVED** [review]

**Review by anonymousp0 — APPROVED** [review]

**Review by Victoriaf1 — APPROVED** [review]

**Comment by EmmyGram1** [comment]
Thank you

**Comment by Ihateu559** [comment]
What is this

**Review by Murtadhashammari — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ Repository | Description
 [AppSec](https://github.com/paragonie/awesome-appsec)								| Resources for learning about application security
 [Asset Discovery](https://github.com/redhuntlabs/Awesome-Asset-Discovery)    | List of resources which help during asset discovery phase of a security assessment engagement
 [Bug Bounty](https://github.com/djadmin/awesome-bug-bounty) 						| List of Bug Bounty Programs and write-ups from the Bug Bounty hunters
-[Capsulecorp Pentest](https://github.com/r3dy/capsulecorp-pentest) 						| Vagrant+Ansible virtual network penetration testing lab. Companion to "The Art of Network Penetration Testing" by Royce Davis
+[Capsulecorp Pentest](https://github.com/r3dy/capsulecorp-pentest) 						| Vagrant+Ansible virtual network penetration testing lab. Companion to "The Art of Network Penetration Testing" by Royce Davis
```
Comment (PAVAN-CHWEET) [inline README.md:17]: Network Penetration Testing is Manually more effective than running a Code. This is done to achieve complete efficiency of the code, So ensuring the main objective is being served and compiling the code is the best way to go with it.

File: README.md [hunk 2]
```diff
@@ -14,7 +14,8 @@ Repository | Description
...
 [Bug Bounty](https://github.com/djadmin/awesome-bug-bounty) 						| List of Bug Bounty Programs and write-ups from the Bug Bounty hunters
-[Capsulecorp Pentest](https://github.com/r3dy/capsulecorp-pentest) 						| Vagrant+Ansible virtual network penetration testing lab. Companion to "The Art of Network Penetration Testing" by Royce Davis
+[Capsulecorp Pentest](https://github.com/r3dy/capsulecorp-pentest) 						| Vagrant+Ansible virtual network penetration testing lab. Companion to "The Art of Network Penetration Testing" by Royce Davis 
+[Celluar Hacking](https://github.com/W00t3k/Awesome-Cellular-Hacking)    | This is a list of hacking research in the 3G/4G/5G cellular security space. 
 [CTF](https://github.com/apsdehal/awesome-ctf) 										| List of CTF frameworks, libraries, resources and softwares
 [Cyber Skills](https://github.com/joe-shenouda/awesome-cyber-skills) | Curated list of hacking environments where you can train your cyber skills legally and safely
 [DevSecOps](https://github.com/devsecops/awesome-devsecops) 						| List of awesome DevSecOps tools with the help from community experiments and contributions
```
Comment (PAVAN-CHWEET) [inline README.md:21]: Software teams use technology to perform automated security testing during development. DevOps teams use it to check the app for security flaws without compromising the delivery timeline. For example, software teams use Amazon Inspectorto automate continual vulnerability management at scale.

File: README.md [hunk 3]
```diff
@@ -14,7 +14,8 @@ Repository | Description
 [AppSec](https://github.com/paragonie/awesome-appsec)								| Resources for learning about application security
 [Asset Discovery](https://github.com/redhuntlabs/Awesome-Asset-Discovery)    | List of resources which help during asset discovery phase of a security assessment engagement
 [Bug Bounty](https://github.com/djadmin/awesome-bug-bounty) 						| List of Bug Bounty Programs and write-ups from the Bug Bounty hunters
-[Capsulecorp Pentest](https://github.com/r3dy/capsulecorp-pentest) 						| Vagrant+Ansible virtual network penetration testing lab. Companion to "The Art of Network Penetration Testing" by Royce Davis
+[Capsulecorp Pentest](https://github.com/r3dy/capsulecorp-pentest) 						| Vagrant+Ansible virtual network penetration testing lab. Companion to "The Art of Network Penetration Testing" by Royce Davis 
+[Celluar Hacking](https://github.com/W00t3k/Awesome-Cellular-Hacking)    | This is a list of hacking research in the 3G/4G/5G cellular security space.
```
Comment (PAVAN-CHWEET) [inline README.md:18]: Phone hacking, being a form of surveillance, is illegal in many countries unless it is carried out as lawful interception by a government agency. In the News International phone hacking scandal, private investigator Glenn Mulcaire was found to have violated the Regulation of Investigatory Powers Act 2000. He was sentenced to six months in prison in January 2007.Renewed controversy over the phone-hacking claims led to the closure of the News of the World in July 2011.

File: README.md (hunk 1 above)
Comment (ronhUs-busrur-7tuxfo) [reply README.md:17]: Awesome thank you so very much for your help

File: README.md (hunk 3 above)
Comment (ronhUs-busrur-7tuxfo) [reply README.md:18]: All set

**Review by ronhUs-busrur-7tuxfo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Sebopela) [reply README.md:17]: Wanna hack someone

File: README.md [hunk 4]
```diff
@@ -14,7 +14,8 @@ Repository | Description
 [AppSec](https://github.com/paragonie/awesome-appsec)								| Resources for learning about application security
```
Comment (Vickybut) [inline README.md:14]: N
```suggestion
 [AppSec](https://github.com/paragonie/awesome-appsec)								| Resources for learning about application security
```

**Review by Vickybut — COMMENTED** [review]
Ok

**Comment by cat998** [comment]
已收到！

**Review by Chan9390 — APPROVED** [review]

**Review by daffroot — COMMENTED** [review]
README.md

**Comment by Matrix1813** [comment]
what

**Comment by Chan9390** [comment]
Closing as the commit is merged.

**Pull request closed without merging**